README.md

```diff
@@ -8,6 +8,10 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/KohlsTechnology/eunomia)](https://goreportcard.com/report/github.com/KohlsTechnology/eunomia)
 [![codecov](https://codecov.io/gh/KohlsTechnology/eunomia/branch/master/graph/badge.svg)](https://codecov.io/gh/KohlsTechnology/eunomia)
 
+## Getting started
+
+If you want to deploy Eunomia from scratch to a local Minikube cluster, begin with the [Getting started](./GETTING_STARTED.md) document. It will guide your through the setup step-by-step.
+
```

Comment:
![Gitops-hierarchy.png](./media/Gitops-hierarchy.png) - "Seeding CR team A" is referenced in 2 boxes, should the second be "Seeding CR team B"?

Reply:
@seanmalloy Do you have the original source of the image? I'd rather not edit the png

Reply:
I believe @Smiley73 may have originally created that...but I'm not positive